### ISSUE TITLE
[sram_ctrl/dv] EQC of tlul_sram_byte

### ISSUE THREAD

**Issue by nasahlpa** [lowRISC/opentitan#23364]
### Description

The SRAM readback feature (c.f., #23212) introduced a couple of changes in the `tlul_sram_byte` module. The feature can be disabled with the SRAM_CTRL.READBACK CSR.
When disabled, the `tlul_sram_byte` module should behave as before this PR. To make sure that this is the case, we should run some equivalence checking (EQC).

**Comment by vogelpi** [comment]
This would be nice to have but I fear we have too many other higher priority things to take care of for TO.

**Comment by vogelpi** [comment]
When readback is disabled, we still have the block-level throughput test which verifies the throughput. I don't think we would gain much by this. Thus moved to M7.

**Comment by vogelpi** [comment]
The block-level throughput test has now been adapted to work w/ and w/o readback being enabled (see https://github.com/lowRISC/opentitan/pull/25145 for details) meaning we have very high confidence that the readback feature "impacts" the design exactly the way intended.

I no longer think we should actually implement the LEC proposed by this issue. This was proposed at the time when we implemented the change to re-risk the tapeout while doing the real DV work in parallel. But since then, we've implemented all remaining tests (thanks @nasahlpa !) and I seriously think we wouldn't gain anything more from implementing this LEC. I am thus closing the issue as not planned and put the issue onto the agenda for the next triaging meeting. Feel free to re-open if you don't agree.

FYI @andreaskurth

**Comment by andreaskurth** [comment]
Agreed, we're closing the coverage of this feature like we do for all other features.
